guava-tests/test/com/google/common/primitives/BytesTest.java

```diff
@@ -119,6 +119,39 @@ public void testLastIndexOf() {
         (byte) 3));
   }
 
+  public void testLastIndexOf_arrayTarget() {
+    assertEquals(-1, Bytes.lastIndexOf(EMPTY, EMPTY));
```

Comment:
`Bytes.indexOf(byte[], byte[])` finds an empty array in an empty array. For symmetry it would make sense for the same to happen when searching from the other direction, i.e. this should be `0` not `-1`.